lib/modules/showImages.js

```diff
@@ -1636,7 +1446,7 @@ addModule('showImages', (module, moduleID) => {
 		}
 		// we need to use a different onload event for videos...
 		if (mediaTag.tagName === 'VIDEO') {
-			mediaTag.addEventListener('loadeddata', handleMediaLoad);
+			mediaTag.addEventListener('loadedmetadata', handleMediaLoad);
```

Comment:
Updated so that expando size is adjusted earlier, as the video element is sized at `loadedmetadata`.